oci/oci.go

```diff
@@ -116,6 +129,23 @@ func (r *Runtime) Name() string {
 // Depending if the container is privileged and/or trusted,
 // this will return either the trusted or untrusted runtime path.
 func (r *Runtime) Path(c *Container) string {
+	if c.runtimeHandler != "" {
+		runtimeHandler, exist := r.runtimes[c.runtimeHandler]
+		if !exist {
+			if c.runtimeHandler == "untrusted" && r.untrustedPath != "" {
```

Comment:
nit: Is there a common variable that could be used over a bare string for untrusted?

Reply:
Yes, I'll add a `const` for that!